pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -380,3 +388,18 @@ func (lp *LifecycleProcessor) setParentIDForMultipleInferredSpans() {
 	lp.requestHandler.inferredSpans[1].Span.ParentID = lp.requestHandler.inferredSpans[0].Span.ParentID
 	lp.requestHandler.inferredSpans[0].Span.ParentID = lp.requestHandler.inferredSpans[1].Span.SpanID
 }
+
+func (lp *LifecycleProcessor) processTrace(spans []*pb.Span) {
+	traceChunk := &pb.TraceChunk{
+		Origin:   "lambda",
```

Comment:
Note: this adds `origin:lambda` to the execution span's trace chunk, which wasn't there previously. However, this is fine because we add that origin later anyway.